apps/oxlint/fixtures/import/.oxlintrc.json

```diff
@@ -0,0 +1,7 @@
+{
+    "plugins": ["import"],
```

Comment:
https://eslint.org/docs/v8.x/use/configure/configuration-files#using-a-configuration-from-a-plugin

The rules for the names are confusing lol:

The eslint-plugin- prefix can be omitted from the plugin name.

Reply:
Not only _can_ it be, I've never seen a config file where it wasn't. I don't even think including that prefix is valid. Either way, I didn't add support for it when parsing this field. Should I add it?

Reply:
nah